core/logger/sugared.go

```diff
@@ -0,0 +1,37 @@
+package logger
+
+// SugaredLogger extends the base Logger interface with syntactic sugar, similar to zap.SugaredLogger.
+type SugaredLogger interface {
+	Logger
+	//TODO document
```

Comment:
missed?

Reply:
Ahh my bad. I added the logs next to implementation so should remove this one.